examples/adyen-dropin-component/app.vue

```diff
@@ -0,0 +1,98 @@
+<script setup lang="ts">
+const { isLoggedIn, login } = useUser();
+const { createOrder } = useCheckout();
+const { refreshSessionContext } = useSessionContext();
+
+// get the login data from nuxt.config.ts
+const {
+  public: { loginData },
+} = useRuntimeConfig();
+
+// useListing for further product search
+const { search, getElements } = useListing({
+  listingType: "productSearchListing",
+});
+// composable used to add to cart a product
+const { addProduct } = useCart();
+// configured apiClient instance in order to make a custom API call
+const { apiInstance } = useShopwareContext();
+// current step for progress bar displaying <ProgressBar /> in a template
+const activeStep = ref(0);
+// container for keeping and displaying an incoming payment state from adyen
+const paymentState = ref();
+
+try {
```

Comment:
It is not good practice to have so many actions in single try/catch block, also console error can be mislead if errors comes `addProduct`.

Reply:
good point, would be enough to just pass error object to console.error? the example should be as short as possible 👀